src/test/codegen/ffi-const.rs

```diff
@@ -0,0 +1,16 @@
+// compile-flags: -C no-prepopulate-passes
+#![crate_type = "lib"]
+#![feature(ffi_const)]
+
+// CHECK-LABEL: @bar()
+#[no_mangle]
+pub fn bar() { unsafe { foo() } }
+
+
+extern {
+    // CHECK-LABEL: @foo() unnamed_addr #1
```

Comment:
Please use pattern matching to match attribute number.

Reply:
Done (I hope, could you check?)

Reply:
What you should’ve made a pattern was the attribute number itself, since that can change due to ordering of the generated code, and thus is liable to flakiness unless properly generalized over.

Something like this:

```
// CHECK-LABEL: @foo(){{.*}}[[FOO_ATTR:#[0-9]+]]
// CHECK: [[FOO_ATTR]] = { ... }
```

Reply:
Should be fixed now. I had to use `CHECK-DAG`!